packages/node_modules/@ciscospark/plugin-people/src/people.js

```diff
@@ -147,7 +153,19 @@ const People = SparkPlugin.extend({
   list(options) {
     if (Array.isArray(options)) {
       const peopleIds = options;
-      return Promise.all(peopleIds.map((personId) => this.batcher.request(personId)));
+      return Promise.all(peopleIds.map((personId) => this.batcher.request(personId)))
+        // remove any not found placeholders
+        // expect notFounds to be the vast minority.
+        // batcher appends the not founds results contiguously after all of the founds.
+        .then((results) => {
+          let idx = results.length - 1;
+          while (idx >= 0 && !results[idx]) {
+            idx -= 1;
+          }
+          return (idx >= 0 && results.slice(results.length - idx)) || [];
+        })
+        // return an empty list on errors
+        .catch(() => []);
```

Comment:
Should we use the logger to log the errors that we are catching?

Reply:
will do